test/sql-tap/index1.test.lua

```diff
@@ -128,11 +128,13 @@ test:do_test(
             val = val * 2
         end
         test:execsql "CREATE INDEX index9 ON test1(cnt)"
-        test:execsql "CREATE INDEX indext ON test1(power)"
-        return test:execsql [[SELECT "name" FROM "_index" WHERE "name"='INDEX9' OR "name"='INDEXT' union SELECT "name" FROM "_space" WHERE "name"='TEST1';]]
+        test:execsql "CREATE INDEX index_t ON test1(power)"
```

Comment:
What is the reason for this renaming?

Reply:
Checkpatch throws an error thinking the word is misspelled.